manifests/config/webui.pp

```diff
@@ -18,33 +18,37 @@
       'G',
     )
 
-    exec { 'semanage-port-http_port_t':
-      command => "semanage port -a -t http_port_t -p tcp ${proxy_https_port}",
-      unless  => "semanage port -l|grep -E \"^http_port_t.*tcp.*${proxy_https_port}\"",
-      path    => ['/bin','/sbin','/usr/bin','/usr/sbin'],
-    }
+    # IPA switched to mod_ssl as the crypto engine for Apache as of version 4.7.0
+    # see https://www.freeipa.org/page/Releases/4.7.0#Highlights_in_4.7.0
+    if versioncmp($facts['ipa_server_version'], '4.7.0') < 0 {
+      exec { 'semanage-port-http_port_t':
```

Comment:
Does this whole block become unnecessary with FreeIPA > 4.7.0? Most of this does not look specific to a crypto library change.